techniques/system/server-common/1.0/reloadRudderServices.st

```diff
@@ -56,6 +60,8 @@ bundle agent system_reload_rudder_services {
       "any" usebundle => service_reload("${relayd_service_name}");
     rudder_server_system_restart_relayd::
       "any" usebundle => service_restart("${relayd_service_name}");
+    rudder_server_system_reload_apache::
```

Comment:
why would apache reload restart cf-serverd?

Reply:
when the network changes